<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1698</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1671</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
border-radius:20px;
border:3px solid #3E2B22;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>841</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
border-radius:20px;
border:3px solid #3E2B22;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>510</y>
      <width>821</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
border-radius:20px;
border:3px solid #3E2B22;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InputImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>821</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-radius:20px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="OutputImage">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>20</y>
      <width>801</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-radius:20px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
color:#D4D7D8;
font-size:25px;

font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Thresholding</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>520</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
color:#D4D7D8;
font-size:25px;

font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Image Segmentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadImage">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>740</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;
	}


QPushButton:hover{
	
	background-color:#998D85;
	}


</string>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>811</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
border-radius:20px;
border:2px solid #D4D7D8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>570</y>
      <width>801</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
border-radius:20px;
border:2px solid #D4D7D8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>580</y>
      <width>20</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
border-radius:20px;
border-right:2px solid #D4D7D8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>580</y>
      <width>20</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
border-radius:20px;
border-right:2px solid #D4D7D8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="local">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>580</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Local Thresholding</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="global">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>630</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Global Thresholding </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Optimal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Optimal Thresholding</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="otsu">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>630</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Otsu</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="spectralone">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Spectral First Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="spectraltwo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>730</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Spectral Second Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="kMeans">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>580</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>K-means</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="regionGrowing">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>630</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Region Growng</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="agglomerative">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>680</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Agglomerative</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="meanShift">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>730</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A29790;
 border-radius: 7px;
 color: #2B313B;
 font-size: 13px;
 font-weight: 700;
border:3px solid #D4D7D8;
 letter-spacing: .8px;
 line-height: 20px;
 text-align: center;
 text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Means Shift</string>
    </property>
   </widget>
   <widget class="QSlider" name="iterationsHorizontalSlider">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>590</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	background:#2B313B;
}
QSlider::groove:horizontal {
    height: 8px;
    background: #2B313B; /* Lighter shade of the dark color */
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background:  #2B313B; /* Mid-tone between the dark and lighter colors */
    border: 2px solid #D4D7D8; /* Matches groove */
    width: 20px;
    height: 20px;
    margin: -6px 0;
    border-radius: 10px;
}

QSlider::sub-page:horizontal {
    background: #A29790; /* Matches handle */
    border-radius: 4px;
}

QSlider::add-page:horizontal {
    background: #2B313B; /* Matches groove */
    border:2px solid #D4D7D8;
    border-radius: 4px;
}
</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>580</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
color:#D4D7D8;
font-size:15px;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>No. Iterations</string>
    </property>
   </widget>
   <widget class="QLabel" name="iterationsLabel">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>590</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2B313B;
color:#D4D7D8;
font-size:15px;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>InputImage</zorder>
   <zorder>OutputImage</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_15</zorder>
   <zorder>label_16</zorder>
   <zorder>local</zorder>
   <zorder>global</zorder>
   <zorder>Optimal</zorder>
   <zorder>otsu</zorder>
   <zorder>spectralone</zorder>
   <zorder>spectraltwo</zorder>
   <zorder>kMeans</zorder>
   <zorder>regionGrowing</zorder>
   <zorder>agglomerative</zorder>
   <zorder>meanShift</zorder>
   <zorder>LoadImage</zorder>
   <zorder>iterationsHorizontalSlider</zorder>
   <zorder>label_21</zorder>
   <zorder>iterationsLabel</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
